util/fpga/splice_rom.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+
+# Utility script to load ROM contents into FPGA bitstream.
+# Usage:
+#   cd $REPO_TOP
+#   ./util/fpga/splice_rom.sh
+#
+# The updated bitstream is located at the same place as the original Vivado bitstream e.g. at
+# $REPO_TOP/build/lowrisc_systems_chip_earlgrey_cw310_0.1/synth-vivado/
+#  lowrisc_systems_chip_earlgrey_cw310_0.1.bit
+#
+# A copy of the original bitstream is created at e.g. at
+# $REPO_TOP/build/lowrisc_systems_chip_earlgrey_cw310_0.1/synth-vivado/
+#  lowrisc_systems_chip_earlgrey_cw310_0.1.bit.orig
+set -e
+
+. util/build_consts.sh
+
+# Change these variables when using the script for a different top level and/or FPGA board.
+TARGET_BOARD="cw310"
+TARGET_TOP="earlgrey"
+TARGET_FILE_EXT=".scr.39.vmem"
+
+TARGET_PREFIX="sw/device/boot_rom"
+TARGET_EXPORT="${TARGET_PREFIX}/boot_rom_export_fpga_${TARGET_BOARD}"
+TARGET="${BIN_DIR}/${TARGET_PREFIX}/boot_rom_fpga_${TARGET_BOARD}"
+
+FPGA_BUILD_DIR=build/lowrisc_systems_chip_${TARGET_TOP}_${TARGET_BOARD}_0.1/synth-vivado
```

Comment:
None of these directory paths are meant to be stable, only the paths in `build-bin` are. Unfortunately, we don't have a build system that moves the bitstream there itself, we do this manually in CI.

So I'm fine with keeping things like that for now, but we'll need to change this if we want to use this script in CI (which is helpful to reduce CI times and test different ROMs)

Reply:
I agree with you. The reason for not changing it now was mainly that I didn't want to change too much stuff at once. The main change to this file is the change of its name. But yes, we should change this in a follow up.